public_suffix_list.dat

```diff
@@ -14373,6 +14373,13 @@ simplesite.pl
 // Submitted by Eddie Jones <eddie@onefoldmedia.com>
 nid.io
 
+// Open Domains : https://open-domains.net
+// Submitted by William Harrison <william@williamharrison.dev>
```

Comment:
Yes, please update.

Reply:
@simon-friedberger What do I need to update here?

Reply:
Sorry, I misread your question here.
> @simon-friedberger Is it possible for me to update the submission email on the PR (currently [admin@open-domains.net](mailto:admin@open-domains.net)) without having our position in the queue affected?
Please update the e-mail here to one that isn't tied to one particular person.
You can also update the e-mail on the PR.

Reply:
```suggestion
// Submitted by William Harrison <admin@open-domains.net>
```